### PR TITLE
refa(components): Remove system helper from <Label>

### PR DESCRIPTION
**Pull request by sebald**

### CODE REVIEW

**Comment by sebald** [comment]
Needs field to be merged first.

**Review by ti10le — DISMISSED** [review]

**Review by ti10le — APPROVED** [review]

File: packages/components/src/Field/Field.tsx [hunk 1]
```diff
@@ -31,21 +31,22 @@ export const Field: React.FC<FieldProps> = ({
...
-        </ValidationMessage>
-      ) : (
-        ''
-      )}
-    </div>
-  );
-};
+    return (
+      <div>
+        <Label htmlFor={htmlFor} className={classNames}>
```
Comment (ti10le) [inline packages/components/src/Field/Field.tsx:36]: classNames is not defined

File: packages/components/src/Field/Field.tsx [hunk 2]
```diff
@@ -31,21 +31,22 @@ export const Field: React.FC<FieldProps> = ({
...
+          {label}
+          {error ? <Required size={16} fill={b2bTheme.colors.red60} /> : ''}
```
Comment (ti10le) [inline packages/components/src/Field/Field.tsx:38]: b2bTheme is not defined

File: packages/components/src/Field/Field.tsx [hunk 3]
```diff
@@ -31,21 +31,22 @@ export const Field: React.FC<FieldProps> = ({
   );
   const errorClassName = useStyles({ color: 'color.red60' });
```
Comment (ti10le) [inline packages/components/src/Field/Field.tsx:32]: never used, labelClassName, too.

**Review by ti10le — APPROVED** [review]

**Comment by ti10le** [comment]
@sebald fyi: i have fixed a lint error but dont know how to handle the other things like in my comments.

**Comment by sebald** [comment]
@ti10le thanks! Looks like I messed up the merge of main.

**Review by ti10le — APPROVED** [review]

**Pull request merged**